crates/matrix-sdk-sqlite/src/crypto_store.rs

```diff
@@ -265,6 +273,18 @@ trait SqliteConnectionExt {
         data: &[u8],
     ) -> rusqlite::Result<()>;
 
+    fn set_direct_withheld(
+        &self,
+        session_id: &[u8],
+        room_id: &[u8],
+        data: &[u8],
+    ) -> rusqlite::Result<()>;
+
+    // fn get_direct_withheld_info(
+    //     &self,
+    //     session_id: Key,
+    //     room_id: Key,
+    // ) -> rusqlite::Result<Option<Vec<u8>>>;
```

Comment:
Commented out function intentionally left in?

Reply:
Removed in 6fd129dcfaa501636e2f23e9381ad2fbad06e0ea.